content/millennium/r4/proprietary-codes.md

```diff
@@ -213,3 +213,14 @@ This code set includes Relationship Types such as Guardian and Emergency Contact
       "display": "Guardian",
       "userSelected": true
     }
+
+#### Code Set 14249 Scheduling Appointment Synonyms or Types
+
```

Comment:
for appointment type we have different code set 14230
Not sure if we should mention appointment type here or not.

Similarly in next line we are saying appointment **type** such as, should we say synonym?

Reply:
We are only exposing 14249 now, do we need to expose both?

Reply:
I'm not asking to expose the other one I'm saying do we have to mention type? Can we just say appointment synonyms?